ch360/documents.go

```diff
@@ -166,24 +172,58 @@ func (client *DocumentsClient) Classify(ctx context.Context, documentId string,
 }
 
 func (client *DocumentsClient) Extract(ctx context.Context, documentId string, extractorName string) (*results.ExtractionResult, error) {
-	response, err := newRequest(ctx, "POST",
-		client.baseUrl+"/documents/"+documentId+"/extract/"+extractorName, nil).
-		issue(client.requestSender)
+	extractResponse := results.ExtractionResult{}
+
+	err := client.extract(ctx,
+		documentId,
+		extractorName,
+		"application/vnd.waives.resultformats.extractdata+json", &extractResponse)
```

Comment:
💪 I think this content type should be defined as a constant on the `ExtractionResult` type.